docker-compose.yaml

```diff
@@ -17,6 +18,7 @@ services:
       SECRET_KEY_BASE: ${SECRET_KEY_BASE}
     ports:
       - "5002:5002"
+      - "9568:9568"
```

Comment:
Do we need this port expose this port? I assume we don't expect any external Prometheus instances to scrape the data? Similar case with port "9090" in the Prometheus service in this `docker-compose.yml`.